libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -227,23 +242,37 @@ void elasticsearch_plugin_impl::doBlock(uint32_t trx_in_block, const signed_bloc
 
 void elasticsearch_plugin_impl::doVisitor(const optional <operation_history_object>& oho)
 {
+   graphene::chain::database& db = database();
+
    operation_visitor o_v;
    oho->op.visit(o_v);
 
    vs.fee_data.asset = o_v.fee_asset;
+   vs.fee_data.asset_name = o_v.fee_asset(db).symbol;
    vs.fee_data.amount = o_v.fee_amount;
+   vs.fee_data.amount_units = (double)(o_v.fee_amount.value)/lookup_pow_table[o_v.fee_asset(db).precision];
```

Comment:
Can remove all (double) casts here

Reply:
had to keep (double) cast in the `fill_price_units` when using the lookup table integrated in the codebase or the program will segfault with SIGFPE, Arithmetic exception.